code/modules/mob/living/carbon/human/human.dm

```diff
@@ -768,7 +768,7 @@
 
 	if(href_list["secrecordComment"])
 		if(hasHUD(usr,"security"))
-			if(usr.incapacitated())
+			if(usr.incapacitated() == !isobserver(usr))
```

Comment:
What is this supposed to be?
Presumably it's supposed to account for ghosts being counted as "incapacitated"?
However, I don't think this is intuitive enough to go without a comment explaining the what/why here

Reply:
I thought that is a check whether one is incapacitated OR a ghost, and if so, render unable to use secrecordComment even if they can see it. Without this ghosts could add comments by clicking the add comment button from the examine info given by sechud.

Reply:
Isn't this one in particular a purely informational topic call? Or do you want to lock ghosts out anyways

Reply:
Oh damn, bear with me, I checked it too quickly yesterday. So actually: the check in question _gives_ the access for ghosts to view the comment log by being able to press that button. While adding comments from there is handled by the next href_list "secrecordadd", into which ghosts do not have access as they are 'incapacitated' as well as same renders them unable to interact with the other buttons present.

Re-adding it with a comment - I guess this is exactly why one should do them comments...